pyblish_maya/lib.py

```diff
@@ -313,3 +310,28 @@ def _show_no_gui():
 
     messagebox.setStandardButtons(messagebox.Ok)
     messagebox.exec_()
+
+
+class Dock(QtWidgets.QWidget):
+
+    def __init__(self, dock_count, parent=None):
+        super(Dock, self).__init__(parent)
+        QtWidgets.QVBoxLayout(self)
+        object_name = "pyblish_maya.dock" + str(dock_count)
+        self.setObjectName(object_name)
+
+
+def dock(window):
+
+    main_window = None
+    for obj in QtWidgets.qApp.topLevelWidgets():
+        if obj.objectName() == "MayaWindow":
+            main_window = obj
+
+    self._dock_count += 1
```

Comment:
I can't reuse the same name, because it steals the widget from a previous dock control and inserts it into a new dock. As the previous behaviour of ```pyblish-lite``` has been to show a new window regardless of whether one already exists, I tried to repeat this by being able to have multiple docks.

It remembers the name from before because I give the widget a specific object name [here](https://github.com/pyblish/pyblish-maya/pull/63/files/4417f2d6e101e357904476de99753ff9d82e16de#diff-dbf2d8dee0a430b4077374d056fbe7d4R321). I could not make it work with not specifying an object name for the widget.